File: Assignments/Assignment_12/p7.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        int count=(row*row)+64;
        for(int i=0;i<row;i++){

                for(int j=0;j<row;j++){

                        if(j%2==0){

                                printf("%c\t",(count));
                                count--;
                        }else{

                                printf("%d\t",(count-64));
                                count--;
                        }
                }
                printf("\n\n");
        }
}

File: JAVA/11.Triangle_Pattern/p7.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
	
		for(int i=row;i>0;i--){
		
			int count=1;
			for(int j=0;j<i;j++){
			
					System.out.print(count+"\t");
					count++;
			}System.out.println();
		}
	}
}
